app_feup/lib/view/Widgets/generic_card.dart

```diff
@@ -123,4 +128,14 @@ class GenericCardState extends State<GenericCard> {
           )
         : null;
   }
+
+  Widget getMoveIcon(context) {
+    return (widget.editingMode != null && widget.editingMode)
+        ? IconButton(
+          iconSize: 22.0,
+          color: Colors.grey.shade500,
+          icon: Icon(Icons.drag_handle_rounded)
```

Comment:
IconButtons required on click action. That's the reason the code do not pass the flutter analyze test.
Use Icon class instead.

`Icon(Icons.drag_handle_rounded, color: Colors.grey.shade500, size: 22.0)`